cogs/economy.py

```diff
@@ -0,0 +1,79 @@
+import discord
+import time
+from discord.ext import commands
+from datetime import datetime
+import math
```

Comment:
NIT: put the math import with the other base imports and add a space before the individual imports